components/brave_wallet_ui/panel/container.tsx

```diff
@@ -358,15 +358,11 @@ function Container (props: Props) {
     }
   }
 
-  const retryHardwareOperation = () => {
-    // signMessageData by default initialized as [{ id: -1, address: '', message: '' }]
-    if (signMessageData && signMessageData.length && signMessageData[0].id !== -1) {
-      onSignData()
-    }
-    if (selectedPendingTransaction) {
-      onConfirmTransaction()
-    }
+  const onSelectTransaction = (transaction: BraveWallet.TransactionInfo) => {
```

Comment:
Did anything change in this file other than moving the function declaration order?

Reply:
Just function declaration order